runtime/pom.xml

```diff
@@ -36,6 +56,11 @@
             <artifactId>ant-junit</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
Why do the dependencies change?

Reply:
We need the SystemUtil class now in the ToolsJarLocator.

I have removed the other dependencies (as far as they are not needed)